File: day 78/homework/3.js
let numbers2 = [1, 3, 5, 7, 9];

let withIndex = numbers.map((num, index) => num + index);

let oddNumbers = withIndex.filter(num => num % 2 !== 0);

let product = oddNumbers.reduce((total, num) => total * num, 1);

console.log("ინდექსდამატებული რიცხვები:", withIndex);
console.log("კენტი რიცხვები:", oddNumbers);
console.log("ნამრავლი:", product);

File: day 27/homework/3.py
def double_numbers(numbers):
    doubled_list = []
    index = 0
    
    # სანამ ინდექსი სიის სიგრძეზე ნაკლებია
    while index < len(numbers):
        doubled_list.append(numbers[index] * 2)
        index += 1 # გადავდივართ შემდეგ ინდექსზე
        
    return doubled_list

# მაგალითი:
print(double_numbers([1, 2, 3, 4]))  # გამონატანი: [2, 4, 6, 8]